src/curriculum/dataflow/dataflow.json

```diff
@@ -101,13 +101,13 @@
                           "name": "Generator",
                           "data": { "generatorType": "sine", "amplitude": 1, "period": 10 },
                           "outputs": [{ "num": [{ "transform-id": "num1" }] }],
-                          "position": [ 250, 300 ]
+                          "position": [ 5, 260 ]
                         },
                         {
                           "id": "transform-id",
                           "name": "Transform",
                           "data": { "transformOperator": "absolute value" },
-                          "position": [ 500, 300 ]
+                          "position": [ 250, 270 ]
```

Comment:
I tweaked these positions so tiles are slightly more presentable when opening side panel (it's been bothering me so I squeezed it into this PR).